scripts/gitlab_cleanup/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+    "context"
+    "fmt"
+    "os"
+    "time"
+
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/client-go/kubernetes"
+    "k8s.io/client-go/rest"
+)
+
+func main() {
+    config, err := rest.InClusterConfig()
+    if err != nil {
+        fmt.Printf("Error creating in-cluster config: %s\n", err)
+        os.Exit(1)
+    }
+
+    clientset, err := kubernetes.NewForConfig(config)
+    if err != nil {
+        fmt.Printf("Error creating clientset: %s\n", err)
+        os.Exit(1)
+    }
+
+    pods, err := clientset.CoreV1().Pods("gitlab").List(context.Background(), metav1.ListOptions{
+        LabelSelector: "app=gitlab,selector=runner",
+    })
+    if err != nil {
+        fmt.Printf("Error listing pods: %s\n", err)
+        os.Exit(1)
+    }
+
+    for _, pod := range pods.Items {
+        if time.Since(pod.CreationTimestamp.Time).Hours() > 12 {
```

Comment:
We could probably extract a const, maybe having it in the env to make this configurable without having to change the code ? Wdyt ?

Reply:
Yes, I think an env var with a default value could be better.